modules/gdscript/gdscript_parser.h

```diff
@@ -433,6 +433,7 @@ class GDScriptParser {
 			OP_CONTENT_TEST,
 			OP_COMP_EQUAL,
 			OP_COMP_NOT_EQUAL,
+			OP_LOGIC_XOR,
```

Comment:
This should be moved higher, after `OP_LOGIC_OR`. Also you should add this to `switch` in the `GDScriptParser::parse_binary_operator()`.